src/app/screens/Home/index.tsx

```diff
@@ -94,6 +100,36 @@ function Home() {
     }
   }
 
+  async function onTabChangeHandler(index: number) {
+    if (index === 1) {
+      await loadInvoices();
+    }
+  }
+
+  async function loadInvoices() {
+    // incoives have already been loaded
+    if (invoiceTransactions) return invoiceTransactions;
+
+    setLoadingInvoices(true);
+    const result = await api.getInvoices();
+
+    const invoices: Transaction[] = result.invoices
+      .filter((invoice) => invoice.settled)
+      .map((invoice) => ({
+        ...invoice,
+        title: invoice.memo,
+        date: dayjs(invoice.settleDate).fromNow(),
+      }));
+
+    for (const invoice of invoices) {
+      const totalAmountFiat = await getFiatValue(invoice.totalAmount);
+      invoice.totalAmountFiat = totalAmountFiat;
```

Comment:
also here maybe a class object with computed attributes would be nice. 

`invoice.totalAmountFiat` could take care of this on its own whenever it is accessed. 
Now we everywhere need to iterate and do those calls manually

Reply:
not sure how this is done in JS in 2022. 

```
class Invoice {

   get totalAmountFiat() {
    return getFiatValue(this.totalAmount);
  }
}
```

but actually I am wondering if we should remove the fiat amounts here? 
If we make fiat enabled/disabled configurable then we would need a check here everywhere and if we actually want the fiat values of the time of the transaction we need to store it in the DB and for the payments that are not in our DB we do not have historic data.

Reply:
Yeah, but we won't tackle this in this PR, right?  
If we want to show historic data we should do a brainstorming.
Lisa also had some comments to this.

Reply:
no. but the more I use it the more I am in favor of removing fiat in the transaction list. wondering what others say.

Reply:
My priorities for the transaction list would be
1. btc with historical fiat prices
2. only btc
3. btc with current fiat prices

Reply:
1. is not possible (we do not have the incoming payments in our DB and we do not have historical rates) - seems breez and zeus do not show amounts. Phoenix and bluewallet have rates (not sure if historical, can't imaging bluewallet has historical rates)

Reply:
Will remove after this is merged. This came in with the original Fiat PR.